### ISSUE TITLE
Color contrast violations on main page

### ISSUE THREAD

**Issue by zaymasood** [pulibrary/pulmap#1000]
### Error in spec/features/accessibility_spec.rb

accessibility when visiting main page complies with WCAG
               
       color-contrast: Elements must have sufficient color contrast (serious)
       https://dequeuniversity.com/rules/axe/4.3/color-contrast?application=axeAPI
       The following 5 nodes violate this rule:
           
               Selector: .col-lg-6.col-12:nth-child(1) > .card > .card-body > .grid-item > p
               HTML: <p>See all records and filter results by dragging, zooming, or searching for locations on the map.</p>
               Fix any of the following:
               - Element has insufficient color contrast of 2.99 (foreground color: #92959a, background color: #ffffff, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1
               
               Selector: .col-lg-6.col-12:nth-child(2) > .card > .card-body > .grid-item > p
               HTML: <p>Detailed building maps of cities and towns created by The Sanborn Map Company.</p>
               Fix any of the following:
               - Element has insufficient color contrast of 2.99 (foreground color: #92959a, background color: #ffffff, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1
               
               Selector: .col-lg-6.col-12:nth-child(3) > .card > .card-body > .grid-item > p
               HTML: <p>Scanned historic, geoscientific, topographic, and bathymetric maps and charts.</p>
               Fix any of the following:
               - Element has insufficient color contrast of 2.99 (foreground color: #92959a, background color: #ffffff, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1
               
               Selector: .col-lg-6.col-12:nth-child(4) > .card > .card-body > .grid-item > p
               HTML: <p>Pixel-based data such as aerial photographs, satellite imagery, land-use maps, and elevation.</p>
               Fix any of the following:
               - Element has insufficient color contrast of 2.99 (foreground color: #92959a, background color: #ffffff, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1
               
               Selector: .center-grid-item > p
               HTML: <p>Data representing geographic features as points, lines, and polygons.</p>
               Fix any of the following:
               - Element has insufficient color contrast of 2.99 (foreground color: #92959a, background color: #ffffff, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1
